models/model_constants.py

```diff
@@ -3,6 +3,9 @@
 TRACKING = 'tracking'
 RE_ID_EVAL = 're-id-eval'
 RE_ID_TRAIN = 're-id-train'
-REID_ACTIONS = [RE_ID_EVAL, RE_ID_TRAIN]
+REID_VIZ = 're-id-viz'
```

Comment:
Change to RE_ID_VIZ. Both to stay consistent with the other three and because in model_runner.py you check `elif args.action == RE_ID_VIZ`